pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -93,7 +93,31 @@ func (m *OSDHealthMonitor) Update(removeOSDsIfOUTAndSafeToRemove bool) {
 }
 
 // checkOSDHealth takes action when needed if the OSDs are not healthy
-func (m *OSDHealthMonitor) checkOSDHealth() error {
+func (m *OSDHealthMonitor) checkOSDHealth() {
+	err := m.checkOSDDump()
+	if err != nil {
+		logger.Debugf("failed to check OSD Dump. %v", err)
```

Comment:
Why not `Errorf`? Was this discussed already?

Reply:
it wasn't discussed, followed the logging level from the code base before refactoring
Below is the existing structure on invoking `checkOSDHealth()`
```
			err := m.checkOSDHealth()
			if err != nil {
				logger.Debugf("failed OSD status check. %v", err)
			}
```
The checkOSDHealth is now refactored to two sub methods, the only difference is the errors are logged inside the sub methods.

Reply:
Makes sense.